doc/sphinxman/source/dftd3.rst

```diff
@@ -222,6 +223,7 @@ available only through the ``DFTD3`` or ``DFTD4`` programs. Once installed, the
 ``dftd3``/|PSIfour| and ``dftd4``/|PSIfour| interfaces are transparent, and all corrections are
 interfaced exactly alike.
 The -D3 interface can use classic or simple-dftd3 programs interchangeably and will prefer the latter.
+Starting in v1.9, the classic program is no longer supported or tested, though it won't be deliberately disabled.
```

Comment:
Probably better since I don't want to promise to never disable the classic. I'll add it to a different PR if that's the only change, since CI is already complete here.